src/dummy.erl

```diff
@@ -1 +1,5 @@
 -module(dummy).
+
+nop() ->
+%%%%%%%%%%%%%%%% Long line comment %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

Comment:
Line 4 is too long: %%%%%%%%%%%%%%%% Long line comment %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%.